web/client-api/src/main/java/io/deephaven/web/client/api/JsPartitionedTable.java

```diff
@@ -81,16 +82,15 @@ public Promise<JsPartitionedTable> refetch() {
 
             return w.getExportedObjects()[0].fetch();
         }).then(result -> connection.newState((c, state, metadata) -> {
-            JsTable keyTable = (JsTable) result;
-            DropColumnsRequest drop = new DropColumnsRequest();
-            drop.setColumnNamesList(new String[] {descriptor.getConstituentColumnName()});
-            drop.setSourceId(keyTable.state().getHandle().makeTableReference());
-            drop.setResultId(state.getHandle().makeTicket());
-            connection.tableServiceClient().dropColumns(drop, metadata, (fail, success) -> {
-                keyTable.close();
+            baseTable = (JsTable) result;
+            SelectOrUpdateRequest view = new SelectOrUpdateRequest();
```

Comment:
Should rebase this off the latest main or merge the latest main so this PR is just the changes on top of that.